text_mining.py

```diff
@@ -0,0 +1,217 @@
+    "less", "ltd", "made", "many", "may", "me", "meanwhile", "might", "mill", "mine", "more", "moreover",
+    "most", "mostly", "move", "much", "must", "my", "myself", "name", "namely", "neither", "never",
+    "nevertheless", "next", "nine", "no", "nobody", "none", "noone", "nor", "not", "nothing", "now", "nowhere",
+    "of", "off", "often", "on", "once", "one", "only", "onto", "or", "other", "others", "otherwise", "our",
+    "ours", "ourselves", "out", "over", "own","part", "per", "perhaps", "please", "put", "rather", "re",
+    "same", "see", "seem", "seemed", "seeming", "seems", "serious", "several", "she", "should", "show",
+    "side", "since", "sincere", "six", "sixty", "so", "some", "somehow", "someone", "something", "sometime",
+    "sometimes", "somewhere", "still", "such", "system", "take", "ten", "than", "that", "the", "their",
+    "them", "themselves", "then", "thence", "there", "thereafter", "thereby", "therefore", "therein",
+    "thereupon", "these", "they", "thick", "thin", "third", "this", "those", "though", "three", "through",
+    "throughout", "thru", "thus", "to", "together", "too", "top", "toward", "towards", "twelve", "twenty",
+    "two", "un", "under", "until", "up", "upon", "us", "very", "via", "was", "we", "well", "were", "what",
+    "whatever", "when", "whence", "whenever", "where", "whereafter", "whereas", "whereby", "wherein",
+    "whereupon", "wherever", "whether", "which", "while", "whither", "who", "whoever", "whole", "whom",
+    "whose", "why", "will", "with", "within", "without", "would", "yet", "you", "your", "yours", "yourself",
+    "yourselves", "the"]
+
+    filtered_word_list = [word for word in word_list if word not in stop_words and word not in stop_words_2]
+
+    return filtered_word_list
+
+
+def get_histogram(word_list):
+    """Takes a list of words as input and returns a dictionary with all the unique
+    words and their word counts
+
+    Args:
+        word_list: a list of words (assumed to all be in lower case with no punctuation)
+
+    Returns:
+        a histogram; a dictionary with all the unique words and their word counts
+    """
+    word_counts = dict()
+
+    for word in word_list:
+        if word != '':
+            word_counts[word] = word_counts.get(word, 0) + 1
+
+    return word_counts
+
+
+def get_top_n_words(word_list, n):
+    """Takes a list of words as input and returns a list of the n most frequently
+    occurring words ordered from most to least frequently occurring.
+
+    Args:
+        word_list: a list of words (assumed to all be in lower case with no punctuation)
+        n: the number of words to return
+
+    Returns:
+        a list of n most frequently occurring words ordered from most to least frequently occurring
+    """
+    word_counts = get_histogram(word_list)
+
+    ordered_by_frequency = sorted(word_counts, key=word_counts.get, reverse=True)
+
+    return ordered_by_frequency[0:n]
```

Comment:
FWIW, you can shorten [0:n] to [:n] and the 0 is implied. Your thing still works, though - just a personal preference thing, really.